client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -173,7 +172,8 @@ const DataFilesToolbar = ({ scheme, api }) => {
   const canRename = getFilePermissions('rename', permissionParams);
   const canMove = getFilePermissions('move', permissionParams);
   const canCopy = getFilePermissions('copy', permissionParams);
-  const canTrash = getFilePermissions('trash', permissionParams);
+  const canTrash =
+    getFilePermissions('trash', permissionParams) && !path.startsWith('.Trash');
```

Comment:
Can you add this additional logic to `canTrash` to the `trash` case of `filePermissions.js` instead?

Reply:
Yeah, that's what I ended up doing to address Wes' similar point above.